From 4ae524d973475da461a092ea969daf4f89c9b69c Mon Sep 17 00:00:00 2001
From: Adeel Arshad <adeel_arshad@mentor.com>
Date: Mon, 10 Oct 2016 16:43:16 +0500
Subject: [PATCH] donot use the local copy of libstdc++

For cross compilation it make more sense to not use the
local copy of precompiled libstdc++ and it also resolves
the build issues of undefined references,
_M_start_thread,
__throw_out_of_range_fmt,
__cxa_throw_bad_array_new_length,
std::logic_error::logic_error(char const*)

Signed-off-by: Adeel Arshad <adeel_arshad@mentor.com>
---
 CodeXL/SConstruct                                   | 1 -
 CodeXL/Utils/AMDTSystemInformationHelper/Sconstruct | 1 -
 2 files changed, 2 deletions(-)

diff --git a/CodeXL/SConstruct b/CodeXL/SConstruct
index 8923020..e742ad3 100755
--- a/CodeXL/SConstruct
+++ b/CodeXL/SConstruct
@@ -65,7 +65,6 @@ initQt4 (CXL_env)
 initTinyXml (CXL_env)
 initQScintilla (CXL_env)
 initQCustomPlot (CXL_env)
-initStdc(CXL_env)
 initJava(CXL_env)
 initBoost(CXL_env)
 
diff --git a/CodeXL/Utils/AMDTSystemInformationHelper/Sconstruct b/CodeXL/Utils/AMDTSystemInformationHelper/Sconstruct
index cff9153..52ec223 100755
--- a/CodeXL/Utils/AMDTSystemInformationHelper/Sconstruct
+++ b/CodeXL/Utils/AMDTSystemInformationHelper/Sconstruct
@@ -46,7 +46,6 @@ initCXLBuild (CXL_env)
 #
 initGtk (CXL_env)
 initTinyXml (CXL_env)
-initStdc(CXL_env)
 initBoost(CXL_env)
 
 # Set the debug/release variant data
-- 
1.9.1

